docs-ref-conceptual/what-is-azure-cli.md

```diff
@@ -1,40 +1,60 @@
-ms.date: 01/30/2020
+ms.date: 06/04/2021
 ms.topic: overview
 ms.service: azure-cli
 ms.devlang: azurecli 
 ms.custom: devx-track-azurecli
 ---
-# What is Azure CLI
+# What is the Azure CLI
 
-The Azure command-line interface (Azure CLI) is a set of commands used to create and manage Azure resources.  The Azure CLI is available across Azure services and is designed to get you working quickly with Azure, with an emphasis on automation.
+The Azure CLI is a cross-platform command-line tool to connect to Azure and execute administrative commands on Azure resources. It allows the execution of commands through a terminal using interactive command-line prompts or a script.
 
-## Key Characteristics
+For interactive use, you first launch a shell such as cmd.exe on Windows, or Bash on Linux or macOS, and then issue a command at the shell prompt. To automate repetitive tasks, you assemble the CLI commands into a shell script using the script syntax of your chosen shell, and then you execute the script.
 
-Azure CLI capabilities make it easy to work with different programing languages and software environments.  For example, Azure CLI:
+You can install the Azure CLI locally on Linux, Mac, or Windows computers. It can also be used from a browser through the Azure Cloud Shell or run from inside a Docker container.
 
-- Is available to install in Windows, macOS, and Linux environments.
+## Syntax samples
 
-- Can also be run in Docker and Azure Cloud Shell.
-- Offers command-line flexibility when managing an Azure solution.
-- Supports long-running operations.
-- Has the ability to use one subscription for all commands, or vary subscriptions per command.
-- Allows for querying of command-line results with query output returned in your format of choice.
-- Has the flexibility to work with multiple clouds.
-- Provides configurable settings for logging, data collection, and default argument values.
-- Is deployed with Resource Manager deployment templates.
+### Subscriptions
+
+The Azure CLI syntax follows a simple `reference name` - `command` - `parameter` - `parameter value` pattern.  For example, switching between subscriptions is often a common task.  Here is the syntax.
+
+```azurecli
+az account set --subscription "my subscription name"
+```
+
+See [Manage subscriptions with Azure CLI](manage-azure-subscriptions-azure-cli.md) to learn about listing, changing, and locking subscriptions.
+
+### Role assignments
+
+Another common use of the Azure CLI is managing role assignments.
+
+```azurecli
+az role assignment create --assignee servicePrincipalName --role Reader
+az role assignment delete --assignee userSign-inName --role Contributor
+```
+
+Now, how easy was that?!  For more information on creating service principals and role assignments, see [Create an Azure service principal with the Azure CLI](create-an-azure-service-principal-azure-cli.md).
 
 ## Current Version
 
 [!INCLUDE [current-version](includes/current-version.md)]
 
+## Prerequisites
```

Comment:
It seems a little odd to have Prerequisites here, but I trust your judgment.

Reply:
@cjgronlund , agreed in the context of how we use this heading in all our docs.  My personal preference is "prepare your environment" and since this isn't a Quckstart, How-to etc., I've switched the heading.  I've also rearranged the H2s as you have to prepare your environment before you run example scripts.